code/index.html

```diff
@@ -2,29 +2,182 @@
+
+    <!-- CSS-LINK -->
+    <link rel="stylesheet" href="./style.css">
+    </head>
+
+    <body>
+    
+    <!-- HEADER VIDEO AND LOGO-->
+    <header>
+      <div id="hero">
+      <video id="heroVideo" loop muted autoplay preload playsinline="auto">
+      <source src="./videos/lefleur-video.mp4">
+
+      Your browser does not support the video tag.
+      </video>
+      <img class="logotype" src="./logo/lefleur-logo.png" />
+      </div>
+    </header>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <!-- MAIN CONTENT-->
+    <main class="main-content">
+      <div class="extra-big-header">
+      Fresh flowers this weekend.
       </div>
+      
+      <div class="selling-copy">
+        <p>Order your beautiful bouquet of flowers before 14:00 and get them delivered the same day. </p>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p> When we send you flowers, we guarantee top quality of fresh flowers such as roses, tulips and seasonal flowers from all our 9700 retail florists. </p>
+
+       <p> – Happiness blooms from within. </p>
       </div>
+
+    <!-- ORDER FORM -->
+    <section class="responsive">
+      <form action="https://httpbin.org/anything" method="post">
+
+        <div class="big-header">Lets order</div>
+
+          <label>Name 
+          <input type="text" id="name" name="name" placeholder="Your name" required>
+          </label> 
+     
+          <br />
+        
+          <label>Email
+          <input type="email" name="email" placeholder="Email address" required>
+          </label>
+     
+          <br />
+        
+         <h3>Choose the size of the bouquet</h3>
+          <select id="size-bouquet" name="size-bouquet" required>
+            <option value="small">Small bouquet, 200 SEK</option>
+            <option value="medium">Medium size bouquet, 300 SEK</option>
+            <option value="large">Large bouquet, 450 SEK</option>
+        </select>
+      
+          <br />
+      
+      <div class="box">
+        <div class="style">
+          
+          <h3>What kind of style do you like?</h3>
+            <input type="radio" id="romantic" name="style" value="romantic">
+            <label for="romantic">Romantic</label>
+        
+          <br />
+
+          <input type="radio" id="classic" name="style" value="classic">
+          <label for="classic">Classic</label>   
+        
+          <br />
+
+          <input type="radio" id="botanical" name="style" value="botanical">
+          <label for="botanical">Botanical</label>
+
+      </div>
+
+      <h3>Would you like to add something?</h3>
+
+      <textarea class="textarea" placeholder="Write here" rows="4" cols="30"></textarea>
```

Comment:
Maybe add `value=""` to the textarea to make sure that the section is on the json form, bescuse now it's not. Not 100% sure if you can add value to textarea since I did not use it in my project.